packages/data/addon/serializers/elide-metadata.ts

```diff
@@ -0,0 +1,181 @@
+/**
+ * Copyright 2020, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ */
+import EmberObject from '@ember/object';
+import { MetricConnection, MetricEdge } from '../models/metadata/metric';
+import { DimensionConnection, DimensionEdge } from '../models/metadata/dimension';
+import { TimeDimensionConnection, TimeDimensionEdge } from '../models/metadata/time-dimension';
+import { TableConnection } from '../models/metadata/table';
+
+type TablePayload = {
+  tables: TableConnection;
+  source: string;
+};
+
+type NormalizedTable = {
+  id: string;
+  name: string;
+  category: string;
+  description: string;
+  cardinality: string;
+  metricIds: string[];
+  dimensionIds: string[];
+  timeDimensionIds: string[];
+  source: string;
+};
+
+type NormalizedColumn = {
```

Comment:
Are these the Payloads used to construct `Metric`, `Dimension`, etc or are they specific to this part? 

It would be nice to move them to their metadata files, so we can use them in the bard-metadata serializer as well